### PR TITLE
Data Interoperability through Standardized Units of Measurement

### PR DESCRIPTION
**Pull request by stuchalk**
This "technique' promotes the use of an Excel spreadsheet that has been populated with chemistry units of measurement based on the guidelines from the IUPAC Green Book, 3rd edition. The premise is that the spreadsheet can be used as a controlled vocabulary for units (and prefixes) allowing unambiguous reporting of the units.

### CODE REVIEW

File: book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+in any accurate and reliable way. {cite:p}`Hanisch2022`
+
+There are many ways in which researchers represent units as they work up their data.  It can be part of the header of
+a column, appended to numeric values as a string, or in some situations implied by the quantity being measured.  The
+common thread here is that units are not usually explicitly (separately) reported and that makes them difficult to deal
+with digitally.  In addition, in many situations a unit is not written in accordance to published guidelines, which in 
+chemistry is the IUPAC "Quantities, Units and Symbols in Physical Chemistry" (or The IUPAC Green Book). {cite:p}`Green2007`
+
+There is a lot of information in the exhaustive tables in the Green Book, and it covers, not surprisingly, a wide 
+range of quantities and units across the many subdisciplines of chemistry. As a guide for humans to read it fine but
```
Comment (Smund27) [inline book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md:42]: it's

File: book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+enabling practical usage in the digital laboratory it does not, where units are almost an afterthought (IUPAC is working
```
Comment (Smund27) [inline book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md:43]: is not

File: book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+The "Example" sheet above, included in the IUPAC Units spreadsheet, demonstrates how to use the unit and prefix lists
+to validate the content of cells. It also uses the VLOOKUP function to then lookup values for the molar masses and the
+factors associated with the prefixes.  This enables the calculation performed in the sheet such that changing the solute 
+and the numeric values for the mass and volume, the final concentration can be calculated.
+
+```{note} Challenge!
+The way that the IUPAC Units spreadsheet is used in the "Example" sheet assumes that the user knows what units are valid
+for a particular quantity.  However, this many not always be true especially if the data is being reused by another
+researcher.  How could the "Example" sheet be upgraded to make the choices available for units based on the quantity?
```
Comment (Smund27) [inline book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md:76]: Is this being posed as homework or we inviting feedback?

Comment (stuchalk) [reply book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md:76]: Yes, this is challenge to the reader to make the spreadsheet even more useful.  I will clarify in an update.

File: book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md (hunk 1 above)
Comment (stuchalk) [reply book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md:42]: Thanks, will fix.

File: book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md (hunk 2 above)
Comment (stuchalk) [reply book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md:43]: Thanks, will fix.

Comment (stuchalk) [reply book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md]: OK fixed in commit 900659894500da82eb08d47a8eb48e7f752f7c15.  Commit message is not what I entered in PyCharm... :(

File: book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md (hunk 1 above)
Comment (stuchalk) [reply book/contributions/tec_iupac_units_excel/techniques/tec_iupac_units_excel.md]: OK fixed in commit 900659894500da82eb08d47a8eb48e7f752f7c15.  Commit message is not what I entered in PyCharm... :(

**Comment by stuchalk** [comment]
I need to change the names of the files.  Other than that is this good to go?  I think I provided all the changes needed...

**Comment by stuchalk** [comment]
I will change the names of the files when I merge into the Cookbook, once one you approves.  If not approved that is fine too we need to keep standards up.

**Pull request merged**